app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -9,23 +10,30 @@ export default Controller.extend({
   application: controller(),
   queryParams: ["period", "order", "asc", "name", "group", "exclude_usernames"],
   period: "weekly",
-  order: "likes_received",
+  order: "",
```

Comment:
We can't have a default value for the query param, because the columns are dynamic. Have to wait until the columns are loaded, then use the first column as the default order